frontend/src/components/ConflictAlert.tsx

```diff
@@ -33,19 +33,33 @@ const ViewText = styled.a`
   text-decoration-line: underline;
   color: #4d6daa;
 `
+type Props = {
+  errorType: 'CONFLICT' | 'EMPTY_FIELD' | 'SERVER_ERROR'
+}
+interface ErrorMessage {
+  [key: string]: string
+}
+const errorMessage: ErrorMessage = {
+  CONFLICT: 'There are conflicts in your booking!',
+  EMPTY_FIELD: 'Oops! There seems to be an empty field.',
+  SERVER_ERROR: 'Server is down now please try again later',
```

Comment:
I think you should include the full stop in this message too!

Reply:
wah good catch! thanksss

Reply:
👍🏼